src/trans/algor/seefmm_mix.F90

```diff
@@ -465,7 +465,8 @@ recursive subroutine comb_xy(kx,px,ky,py,kxy,pxy,kindex)
 integer(kind=jpim), intent(in)  :: kxy
 real(kind=jprb),    intent(out) :: pxy(:)
 integer(kind=jpim), intent(out) :: kindex(:)
-integer(kind=jpim) :: jxy,ix,iy,iret
+integer(kind=jpim) :: iret
+!integer(kind=jpim) :: jxy
```

Comment:
I suppose you kept this variable in a comment because it's referenced in some commented-out code below?

Reply:
Confirmed